src/org/jitsi/impl/neomedia/rtp/translator/SimulcastController.java

```diff
@@ -147,6 +147,20 @@ public long getTargetSSRC()
      */
     public boolean accept(byte[] buf, int off, int len)
     {
+        int targetIndex = bitstreamController.getTargetIndex()
+            , currentIndex = bitstreamController.getCurrentIndex();
+
+        if (targetIndex < 0 && currentIndex > -1)
+        {
+            synchronized (this)
```

Comment:
sorry, i answered my own question and see that we're copying over the `targetIndex` value from the old one when we do `new BitstreamController(bitstreamController, -1);`.  i see now.